eml_parser/decode.py

```diff
@@ -118,8 +118,14 @@ def decode_string(string: bytes, encoding: typing.Optional[str]) -> str:
 
     if chardet:
         enc = chardet.detect(string)
-        if not (enc['confidence'] is None or enc['encoding'] is None) and not (enc['confidence'] == 1 and enc['encoding'] == 'ascii'):
-            value = string.decode(enc['encoding'], 'replace')
+        if enc['encoding'] and enc['encoding'].lower() == 'viscii':
+            value = string.decode('ascii', 'replace')
```

Comment:
Please add a short comment under the conditional explaining why this is required (chardet detects it as viscii... python does not support viscii)... that helps to understand the reason behind this at a later time :-)